core/src/main/scala/com/nvidia/spark/rapids/tool/profiling/ProfileOutputWriter.scala

```diff
@@ -51,27 +51,43 @@ class ProfileOutputWriter(outputDir: String, filePrefix: String, numOutputRows:
     }
   }
 
+  def write(headerText: String, outRows: Seq[ProfileResult],
+      emptyTableText: Option[String] = None, tableDesc: Option[String] = None): Unit = {
+    writeTextTable(headerText, outRows, emptyTableText, tableDesc)
+    ProfileOutputWriter.writeCSVTable(headerText, outRows, outputCSV, outputDir)
+  }
+
+  def close(): Unit = {
+    textFileWriter.close()
+  }
+}
+
+object ProfileOutputWriter {
+  val CSVDelimiter = ","
+
   private def stringIfempty(str: String): String = {
     if (str == null || str.isEmpty) "\"\"" else str
   }
 
-  private def writeCSVTable(header: String, outRows: Seq[ProfileResult]): Unit = {
+  /**
+   * Write a CSV file give the input header and data.
+   */
+  def writeCSVTable(header: String, outRows: Seq[ProfileResult],
+      outputCSV: Boolean = true, outputDir: String): Unit = {
```

Comment:
The `outputCSV: Boolean = true` should be removed as an argument from this method. 
The method's name is  `writeCSVTable` which makes the argument to writeCSV redundant.
The method should simply prints to CSV without checks. the caller is responsible to do that check.

Reply:
removed `outputCSV` from `writeCSVTable`